data_safe_haven/resources/workspace/ansible/host_vars/localhost.yaml

```diff
@@ -13,6 +13,7 @@ package_categories:
       - clamav
       - clamav-daemon
       - clamav-unofficial-sigs
+      - ubuntu-drivers-common
```

Comment:
I don't think this is available for e.g. focal. Can you check?

Reply:
It might not be, but we don't support focal, only jammy.